validate-csp

```diff
@@ -33,7 +33,9 @@ OptionParser.new do |opts|
 end.parse!
 
 # Determine whether we are working a URL or a string.
-if ARGV[0].match(/^http/)
+if ARGV.length == 0
```

Comment:
This feels icky. Let me have a look at `ARGV` and `ARGF` because there needs to be a nicer and more robust way of handling this.